src/main/java/stepsdesign/CdfHubSteps.java

```diff
@@ -88,6 +92,35 @@ public static void verifyPluginIsDeleted() {
   }
   @Then("Click on {string} button")
   public static void clickOnButton(String buttonName) {
-    CdfHubActions.clickOnHomePage(buttonName);
+    CdfHubActions.clickOnButton(buttonName);
+  }
+  @Then("Verify that {string} plugin is successfully verify in studio")
+  public static void verifyPluginInStudio(String pluginName) {
+    CdfHubActions.verifyPluginInStudio(pluginName);
+  }
+  @Then("Verify that {string} plugin is displayed on the Hub page")
+  public static void verifyPluginIsDisplayed(String pluginName) {
+    CdfHubActions.clickOnPlugin(pluginName);
+  }
+  @Then("Create the pipeline")
+  public static void createPipeline() {
+    CdfHubActions.clickOnCreateButton();
+    CdfHubActions.clickOnFinish();
+  }
+  @Then("Click on the Hamburger bar on the left panel")
+  public static void clickOnTheHamburgerIcon() {
+    CdfHubActions.clickOnHamburgerMenu();
+  }
+  @Then("Click on the {string} from the left panel")
+  public static void clickOnTheSideOptions(String option) {
```

Comment:
Its hamburger menu option

Reply:
it is not for hamburger menu it is for the options inside hamburger so i have change it to **clickOnOptionFromHamburgerMenu**